modules/web/src/app/core/components/sidenav/template.html

```diff
@@ -28,21 +28,61 @@
       <span *ngIf="!isSidenavCollapsed()">Project Overview</span>
     </a>
   </div>
+  <ng-container *ngIf="!isSidenavCollapsed()">
+    <km-side-nav-field label="Resources"
+                       icon="km-icon-cluster">
+      <div class="sub-menu-item"
+           [ngClass]="{'active': checkUrl(view.Clusters), 'km-disabled': !getMenuItemClass(view.Clusters)}">
+        <a id="km-nav-item-clusters"
+           fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.Clusters)">
+          <span>Clusters</span>
+        </a>
+      </div>
 
-  <div [matTooltip]="getTooltip(view.Clusters)"
-       [ngClass]="getMenuItemClass(view.Clusters)"
-       class="menu-item">
-    <a id="km-nav-item-clusters"
-       fxLayoutAlign=" center"
-       [routerLink]="getRouterLink(view.Clusters)"
-       [ngClass]="getLinkClass(view.Clusters)">
-      <i class="km-icon-cluster"></i>
-      <span *ngIf="!isSidenavCollapsed()">Clusters</span>
-    </a>
-  </div>
+      <div class="sub-menu-item"
+           [ngClass]="{'active': checkUrl(view.ExternalClusters), 'km-disabled': !getMenuItemClass(view.ExternalClusters)}">
+        <a id="km-nav-item-external-clusters"
+           fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.ExternalClusters)">
+          <span>Extrnal Clusters</span>
+        </a>
+      </div>
+    </km-side-nav-field>
+  </ng-container>
+
+  <ng-container *ngIf="isSidenavCollapsed()">
```

Comment:
![mobile-view](https://user-images.githubusercontent.com/17727069/201865551-8fd0b6e8-a890-4a34-91ab-d71a1eb72e20.png)


- Move parallel to icon see in ticket and see in Darkmode too related